requirements.txt

```diff
@@ -3,3 +3,4 @@ setuptools
 six
 metaextract
 typing;python_version<'3.5'
+ddt
```

Comment:
wrong. it's only a test requirement and already listed in `test-requirements.txt`

Reply:
Oops, my test config was wrong.